db/structure.sql

```diff
@@ -106,7 +106,7 @@ CREATE TYPE public.review_state AS ENUM (
 
 SET default_tablespace = '';
 
-SET default_table_access_method = heap;
+SET default_with_oids = false;
```

Comment:
What does this do?

Reply:
I couldn't tell. This file is generated by Rails every time I recreate the migrations.

Reply:
I am on your branch and I ran the migrations on my local and I don't have that 🤔 
![image](https://user-images.githubusercontent.com/37642427/84047553-cf2e1300-a981-11ea-980f-f1eba3699c09.png)

Reply:
This is most likely a DB version thing.

@mrubi-rootstrap which version of Postgres do you have?

Reply:
It seems like a Postgres setting, yes.
I am using postgres:11 in a docker container, will look for setting that value to match the previous value

Reply:
In the mean time I reverted it to its previous value